Multisig/Logic/Models/PrivateKey/web3swift/BIP39.swift

```diff
@@ -117,7 +117,7 @@ public class BIP39 {
     static public func mnemonicsToEntropy(_ mnemonics: String,
                                           language: BIP39Language = BIP39Language.english) -> Data? {
         let wordList = mnemonics.components(separatedBy: " ")
-        guard wordList.count >= 12 && wordList.count.isMultiple(of: 4) else {return nil}
+        guard wordList.count >= 12 && wordList.count.isMultiple(of: 3) else {return nil}
```

Comment:
really? wasn't it the library code? are we sure this is valid?

Reply:
maybe comment/ explanation or extracting this into parameter/constant would help

Reply:
yes, this is a bug in the lib. I created an issues there and a pr to fix it https://github.com/matter-labs/web3swift/pull/301

I think there is no need in a separate variable.